examples/remote_execution/buildbarn/.buckconfig

```diff
@@ -2,13 +2,10 @@
 root = .
 prelude = prelude
 
-[buck2]
-digest_algorithms = SHA256
-
 [buck2_re_client]
-action_cache_address = http://localhost:8980
-engine_address = http://localhost:8980
-cas_address = http://localhost:8980
+action_cache_address = grpc://localhost:8980
```

Comment:
I'd rather keep this as `http://`. 

Neither of those are actually valid as per [the GRPC spec for URIs](https://github.com/grpc/grpc/blob/master/doc/naming.md), but `grpc://` appears to be more used for when TLS is desired, so I intend to change #174 to respect that (at which point this would break)

Reply:
Actually, let's keep this as-is, and add a separate config field for TLS. I'll update #156 with that.